src/components/modules/Instagram/index.js

```diff
@@ -100,10 +99,8 @@ export default class Instagram extends PureComponent {
     return (
       <Responsive>
         {({ minWidth, width }) => {
-          const showMobileLayout = !minWidth(BREAKPOINTS.LG);
-          return showMobileLayout
-            ? this.renderMobile(width)
-            : this.renderDesktop();
+          const isDesktop = minWidth(BREAKPOINTS.LG);
+          return isDesktop ? this.renderDesktop() : this.renderMobile(width);
```

Comment:
this is just a philosophical question, but I was wondering why you kept the desktop and mobile in a single component vs creating separate InstagramDesktop and InstagramMobile components.

Reply:
Good question

When I was first hooking up the responsive, I was distinguishing them wholesale.

There was a lot of duplications both with markup + styles, but also component props + responsive render props, in some cases.

Still evolving my feelings on ergonomic best practices. LMK if you have thoughts